api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -516,6 +523,37 @@ message HttpConnectionManager {
   bool strip_matching_host_port = 39;
 }
 
+// The configuration to customize local reply returned by Envoy.
+message LocalReplyConfig {
+  // Configuration of list of mappers which allows to filter and change local response.
+  // The mappers will be checked by the specified order until one is matched.
+  repeated ResponseMapper mappers = 1;
+
+  // The configuration to form response body from the :ref:`command operators <config_access_log_command_operators>`
+  // and to specify response content type as one of: plain/text or application/json.
+  // If the matched `mapper <envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.LocalReplyConfig.mapper>`
+  // has a valid `format <envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.ResponseMapper.body_format>`,
+  // it will be used, this one will not be used.
+  config.core.v3.SubstitutionFormatString body_format = 2;
```

Comment:
It might be clearer to call this `default_body_format` or `fallback_body_format`.

Reply:
I prefer the top level one as `body_format`,  and the one inside `mapper` as `body_format_override` as most people will just use the default `body_format`, only some users need to use per-mapper one to override it.